modules/ocf_hpc/files/add_slurm_users

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+'''
+Script that adds users in the 'ocfhpc' LDAP group to SLURM as a SLURM user.
+'''
+import subprocess
+
+from ocflib.account import search
+from ocflib.account import utils
+
+DEFAULT_ACCOUNT = 'users'
+
+
+def main():
+    ocfhpc_ldap_user_set = set(utils.list_group('ocfhpc'))
+
+    # This produces a newline-separated list of SLURM users.
+    ocfhpc_slurm_user_list = subprocess.check_output(
+        ('sacctmgr', '-noP', 'list', 'users', 'format=User'),
+        universal_newlines=True,
```

Comment:
Elsewhere in the codebase when this pattern occurs, we don't specify `universal_newlines` but rather `decode('utf-8')` the result. It may be better to maintain consistency. I'll leave it up to your discretion.